src/components/chart/ha-chart-base.ts

```diff
@@ -148,17 +150,40 @@ export class HaChartBase extends LitElement {
           });
         }
       };
+
+      // Mobile devices don't trigger mouse leave events to hide the tooltip,
+      // instead, hide it whenever the user touches outside of the chart
+      const handleTouchStart = (_ev: TouchEvent) => {
+        // NOTE: if the touch event took place on the chart, echarts will show
+        // the tooltip. Hence, there's no need to check that the event is
+        // within the bounds of the chart
+        this.chart?.dispatchAction({
+          type: "hideTip",
+        });
+      };
+
       window.addEventListener("keydown", handleKeyDown);
       window.addEventListener("keyup", handleKeyUp);
+      window.addEventListener("touchstart", handleTouchStart);
       this._listeners.push(
         () => window.removeEventListener("keydown", handleKeyDown),
-        () => window.removeEventListener("keyup", handleKeyUp)
+        () => window.removeEventListener("keyup", handleKeyUp),
+        () => window.removeEventListener("touchstart", handleTouchStart)
       );
     }
   }
 
   protected firstUpdated() {
     this._setupChart();
+
+    // Prevent scrolling the page on mobile devices when "dragging" the chart
+    // to move the tooltip
+    const handleTouchStart = (_ev: TouchEvent) => {
+      _ev.preventDefault();
+      _ev.stopPropagation();
+    };
+
+    this._chartContainer?.addEventListener("touchstart", handleTouchStart);
```

Comment:
This disables scrolling and the chart can take the whole screen on mobile, leaving the user with no way to scroll

Reply:
I guess this could be worked around by disabling the addition when in fullscreen.